src/operator/merge.ts

```diff
@@ -2,6 +2,15 @@ import {Observable} from '../Observable';
 import {merge as mergeStatic} from './merge-static';
 import {Scheduler} from '../Scheduler';
 
+/**
+ * Flattens an Iterable of Observables into one Observable, without any transformation.
```

Comment:
Isn't this description for `mergeAll`? `merge` itself could be explained with "creates a result Observable which emits values from every given input Observable".